examples/gradle-groovy-dsl/diktat-analysis.yml

```diff
@@ -282,9 +282,19 @@
 # Use extra functions for it instead.
 - name: CUSTOM_GETTERS_SETTERS
   enabled: true
+# Checks if null-check was used explicitly (for example: if (a == null))
```

Comment:
as examples right now stay in the same repo with the main diktat - may be we will copy `diktat-analysis.yml` from main repo somehow?

Reply:
I've added it to `release.yml`, since we run functional tests on release versions.